CONTRIBUTING.md

```diff
@@ -154,6 +154,7 @@ Make sure that [Travis](http://www.travis-ci.org) greenlights the pull request w
  - use lower case with underscores for method names
  - it is generally preferred to use ASCII operators and identifiers over
    Unicode equivalents whenever possible
+ - function signature should not be followed by an empty line
```

Comment:
Yeah, I was skeptical with the suggestion too.

The current diff LGTM, can go ahead with it :+1: